site/content/en/docs/tutorials/configuring_creds_for_aws_ecr.md

```diff
@@ -61,6 +61,9 @@ Enable the minikube registry-creds addon with the following command:
 minikube addons enable registry-creds
 ```
 
+The add-on will create start and create secrets in each namespace that can be used for `imagePullSecrets` on a pod.  It also updates the `default` service account in each namespace to use these `imagePullSecrets`, which means any pod created without an explicit service account other than `default`
```

Comment:
The `create start` path is throwing me off, is `create` supposed to be removed?

Reply:
Ah whoopsies!  Fix incoming!